inst/tests/tests.Rraw

```diff
@@ -16774,3 +16774,18 @@ rm(s1, s2, class2132)
 ########################
 #  Add new tests here  #
 ########################
+
+# round, trunc, floor, and ceiling should all be 'integer' and and have class 'ITime'
+DT = data.table(time = as.ITime(seq(as.POSIXct("2020-01-01 07:00:00"), as.POSIXct("2020-01-01 09:00:00"), "67 sec")))
+
+DT[, ':='(
+  round_hour = round(time, "hour"),
+  round_min = round(time, "min"),
+  trunc_hour = trunc(time, "hour"),
+  trunc_min = trunc(time, "min"),
+  floor_hour = floor(time),
+  ceiling_hour = ceiling(time)
+)]
+
+test(2133.1, TRUE, all(sapply(DT, function(x) class(x) == "ITime")))
```

Comment:
you should be testing for the _values_ as well, not just the output type.

please also include a test that touches on the rounding-at-30 issue you mentioned as well so that we can detect any changes to behavior in the future